### DISCUSSION TITLE
I would like to contribute

### DISCUSSION THREAD

**Discussion by FerhatSaritas** [quran/quran.com-frontend-next#1945] [General]
Salam,
I am working with React and Next professionally and would love to contribute to this project. As far as I know, all issues under `good first issue` are taken or done. Is there anything else I could work on? Perhaps a bug? I asked on the discord but got no answer.

**Comment by osamasayed** [comment]
Wa'likum Alsalam, what is your handle on Discord?

**Reply by FerhatSaritas** [reply]
Ferhat #5219
